src/languages/en-US/errors.json

```
@@ -1,4 +1,13 @@
 {
+	"genericUnknownChannel": "I'm sorry, I tried to operate on a channel unknown to Discord and failed, this error has been reported to the developers.",
+	"genericUnknownGuild": "I'm sorry, I tried to operate on a server unknown to Discord and failed, this error has been reported to the developers.",
+	"genericUnknownMember": "I'm sorry, I tried to operate on a member unknown to Discord and failed, this error has been reported to the developers.",
+	"genericUnknownMessage": "I'm sorry, I tried to operate on a message unknown to Discord and failed, this error has been reported to the developers.",
+	"genericUnknownRole": "I'm sorry, I tried to operate on a role unknown to Discord and failed, this error has been reported to the developers.",
+	"genericMissingAccess": "I'm sorry, I tried to operate on a resource without the necessary permissions and failed, this error has been reported to the developers.",
```

```suggestion
	"genericUnknownChannel": "I'm sorry, I tried to perform an action for a channel unknown to Discord and failed, this error has been reported to the developers.",
	"genericUnknownGuild": "I'm sorry, I tried to perform an action for a server unknown to Discord and failed, this error has been reported to the developers.",
	"genericUnknownMember": "I'm sorry, I tried to perform an action for a member unknown to Discord and failed, this error has been reported to the developers.",
	"genericUnknownMessage": "I'm sorry, I tried to perform an action for a message unknown to Discord and failed, this error has been reported to the developers.",
	"genericUnknownRole": "I'm sorry, I tried to perform an action for a role unknown to Discord and failed, this error has been reported to the developers.",
	"genericMissingAccess": "I'm sorry, I tried to perform an action for a resource without the necessary permissions and failed, this error has been reported to the developers.",
```

While `operate on` is technically correct, it's more commonly related to surgical operations.